BotModules/SteamBot/index.js

```diff
@@ -6,11 +6,15 @@ module.exports = function(bot) {
   var commands = [{
     keyword: '/games',
     helptext: 'Sends a list of all shared games between mentioned users',
-    action: pullMentions(ShowAllSharedGames)
+    action: ShowAllSharedGames
   },{
     keyword: '/pickgame',
     helptext: 'Sends a game chosen randomly from a list of games shared between mentioned users',
-    action: pullMentions(PickRandomSharedGame)
+    action: PickRandomSharedGame
+  },{
+    keyword: '/search',
+    helptext: 'Searches Games Collection for game that most closely matches message content. Limits results up to 3',
+    action: SearchGames
```

Comment:
One change to this, could we set preserve for this, I think its helpful to see your query as well as the results. Just a personal preference.

Reply:
Fixed by changing preserve to autoCleanup and reversing the default behavior. Now the commander only removed the command if you explicitly tell it to clean. 

Changed in 8956031dab6a1252d5aa0241e88522f6c953fe77